src/utils.ts

```diff
@@ -64,6 +79,11 @@ export function isUrl(input: string): boolean {
   return /^https?:\/\//.test(input);
 }
 
+// Check whether the input is a did
```

Comment:
In comment strings, we can be precise with the acronyms, eg. `DID` here.

Reply:
27e9a28f819fb09a9c4ea3796d96362b28f89658 solves this